code/game/objects/items/devices/scanners.dm

```diff
@@ -253,9 +253,29 @@ GENE SCANNER
 		var/mob/living/carbon/C = M
 		var/list/damaged = C.get_damaged_bodyparts(1,1)
 		if(length(damaged)>0 || oxy_loss>0 || tox_loss>0 || fire_loss>0)
-			to_chat(user, "<span class='info'>\tDamage: <span class='info'><font color='red'>Brute</font></span>-<font color='#FF8000'>Burn</font>-<font color='green'>Toxin</font>-<font color='blue'>Suffocation</font>\n\t\tSpecifics: <font color='red'>[brute_loss]</font>-<font color='#FF8000'>[fire_loss]</font>-<font color='green'>[tox_loss]</font>-<font color='blue'>[oxy_loss]</font></span>")
-			for(var/obj/item/bodypart/org in damaged)
-				to_chat(user, "\t\t<span class='info'>[capitalize(org.name)]: [(org.brute_dam > 0) ? "<font color='red'>[org.brute_dam]</font></span>" : "<font color='red'>0</font>"]-[(org.burn_dam > 0) ? "<font color='#FF8000'>[org.burn_dam]</font>" : "<font color='#FF8000'>0</font>"]")
+			var/dmgreportlist = "<pre><font face='Verdana'>&#9;\
+							<font color='#0000CC'>Damage:</font>&#9;&#9;\
+							<font color='red'><b>Brute</font>&#9;\
+							<font color='orange'>Burn</font>&#9;\
+							<font color='green'>Toxin</font>&#9;\
+							<font color='purple'>Suffocation</b></font><br>\
+
+							&#9;&nbsp;&nbsp;&nbsp;&nbsp;\
+							<font color='#0000CC'>Overall:</font>&#9;\
+							<font color='red'>[brute_loss]</font>&#9;\
+							<font color='orange'>[fire_loss]</font>&#9;\
+							<font color='green'>[tox_loss]</font>&#9;\
+							<font color='purple'>[oxy_loss]</font><br>"

```

Comment:
Something about a `<pre>` full of tab characters seems iffy, we have `<table>` available